<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoorMachineDialog</class>
 <widget class="QDialog" name="DoorMachineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="MyLineEdit" name="lineEditName1">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>127</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>name1</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditNetName1">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>127</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>netname1</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DoorMachine_2">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>73</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DoorMachine_3">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>73</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址网络名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditNetName2">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>181</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>netname2</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditName2">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>181</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>name2</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditName3">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>235</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>name3</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditNetName3">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>235</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>netname3</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditNetName4">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>289</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>netname4</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditName4">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>289</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>name4</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditNetName5">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>343</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>netname5</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditName5">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>343</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>name5</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditName6">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>397</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>name6</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frameDoorMachineSetting">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>580</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditNetName6">
   <property name="geometry">
    <rect>
     <x>432</x>
     <y>397</y>
     <width>268</width>
     <height>54</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>netname6</string>
   </property>
   <property name="maxLength">
    <number>49</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DoorMachine_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>289</y>
     <width>164</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DoorMachine_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>397</y>
     <width>164</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DoorMachine_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>181</y>
     <width>164</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DoorMachine_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>235</y>
     <width>164</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DoorMachine_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>343</y>
     <width>164</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DoorMachine_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>127</y>
     <width>164</width>
     <height>54</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>地址1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>10</y>
     <width>510</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>门口机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>frameDoorMachineSetting</zorder>
  <zorder>lineEditName1</zorder>
  <zorder>lineEditNetName1</zorder>
  <zorder>label_DoorMachine_2</zorder>
  <zorder>label_DoorMachine_3</zorder>
  <zorder>lineEditNetName2</zorder>
  <zorder>lineEditName2</zorder>
  <zorder>lineEditName3</zorder>
  <zorder>lineEditNetName3</zorder>
  <zorder>lineEditNetName4</zorder>
  <zorder>lineEditName4</zorder>
  <zorder>lineEditNetName5</zorder>
  <zorder>lineEditName5</zorder>
  <zorder>lineEditName6</zorder>
  <zorder>lineEditNetName6</zorder>
  <zorder>label_DoorMachine_6</zorder>
  <zorder>label_DoorMachine_8</zorder>
  <zorder>label_DoorMachine_4</zorder>
  <zorder>label_DoorMachine_5</zorder>
  <zorder>label_DoorMachine_7</zorder>
  <zorder>label_DoorMachine_1</zorder>
  <zorder>label_title</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditName1</tabstop>
  <tabstop>lineEditNetName1</tabstop>
  <tabstop>lineEditName2</tabstop>
  <tabstop>lineEditNetName2</tabstop>
  <tabstop>lineEditName3</tabstop>
  <tabstop>lineEditNetName3</tabstop>
  <tabstop>lineEditName4</tabstop>
  <tabstop>lineEditNetName4</tabstop>
  <tabstop>lineEditName5</tabstop>
  <tabstop>lineEditNetName5</tabstop>
  <tabstop>lineEditName6</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
